enterprise/ha/src/test/java/org/neo4j/kernel/ha/HaIPv6ConfigurationTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.ha;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.neo4j.cluster.ClusterSettings;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.factory.HighlyAvailableGraphDatabaseFactory;
+import org.neo4j.test.rule.TestDirectory;
+
+public class HaIPv6ConfigurationTest
+{
+    @Rule
+    public TestDirectory dir = TestDirectory.testDirectory();
+
+    @Test
+    public void testClusterWithLocalhostAddresses() throws Throwable
+    {
+        GraphDatabaseService db = new HighlyAvailableGraphDatabaseFactory()
+                .newEmbeddedDatabaseBuilder( dir.makeGraphDbDir() )
+                .setConfig( ClusterSettings.cluster_server, "[::1]:5000" )
+                .setConfig( ClusterSettings.initial_hosts, "[::1]:5000" )
+                .setConfig( HaSettings.ha_server, "[::1]:6000" )
+                .setConfig( ClusterSettings.server_id, "1" )
+                .newGraphDatabase();
+
```

Comment:
I know you're just making sure there is no crash or such, but could you add something which actually uses the database? Create a node or something basic, just to really really verify that it works